backend/tests/load_testing/user_behaviors/base.py

```
@@ -0,0 +1,158 @@
+            endpoint,
+            json=json_data,
+            headers=self._get_auth_headers(),
+            name=name or endpoint,
+            catch_response=True,
+            **kwargs
+        ) as response:
+            if response.status_code in (200, 201):
+                response.success()
+                try:
+                    return response.json()
+                except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)